v4/source/saml.yaml

```diff
@@ -27,6 +27,15 @@
         ##### Permissions
         No permission required.
       operationId: GetSamlMetadataFromIdp
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                saml_metadata_url:
+                  type: string
+                  description: The URL from which to retrieve the Saml IDP data.
```

Comment:
```suggestion
                  description: The URL from which to retrieve the SAML IDP data.
```

Reply:
This is an acronym (not an initialism), just confirming... do we use all caps for those?

Reply:
Thanks. I guess I was thinking of [British English](https://en.wikipedia.org/wiki/Capitalization_in_English#Capitalization_of_acronyms_and_initialisms).